starter/build.gradle

```diff
@@ -2,7 +2,11 @@ plugins {
     id "java"
 }
 
-apply plugin: 'org.springframework.boot'
+repositories {
+    jcenter()
```

Comment:
leave just this one, jitpack is not used here, and maven central already covered in jcenter. Also better to move repo section to the parent build.gradle to subprojects. As well as target/source compatibility